src/sfctl/commands.py

```diff
@@ -203,6 +203,12 @@ def load_command_table(self, args): #pylint: disable=too-many-statements
                 group.command('command', 'invoke_infrastructure_command')
                 group.command('query', 'invoke_infrastructure_query')
 
+            with super_group.group('property') as group:
```

Comment:
You should add scenario tests for this, take a look at the existing tests to see how to write scenario tests and the [wiki documentation](https://github.com/Azure/service-fabric-cli/wiki/Testing)

Reply:
I had a look at writing some scenario tests - the only Service Fabric name that I can guarantee is in the cluster is the 'System' name. I can test the 'list' and 'get' against it but it's restricted as far as 'put' and 'delete'. I can either deploy an application using the 'sfctl application' commands or update this PR to include a new property group for 'name' (i.e. sfctl name create "{\"Name\":\"TestName\"}") and couple the property tests to the name commands. How do you suggest I proceed? Docs on this API are here: https://docs.microsoft.com/en-us/rest/api/servicefabric/sfclient-index-property-management